styles/instagram/catppuccin.user.css

```diff
@@ -112,6 +112,9 @@
       scrollbar-width: none;
       scrollbar-color: @accent-color @crust;
     } // Text selection
+    input::placeholder {
+      color: @text !important;
+    }
```

Comment:
Hmm probably should be subtext0 or something? A little dimmer than the text color ideally.

Reply:
i think i may be missing something, as it doesn't seem to be themed